app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::GreetingsController < ApplicationController
+  def random
+    greetings = Greeting.all
+    greeting = greetings.map(&:message)
+    render json: greeting.sample
+  end
```

Comment:
- [ ] Great job so far but I think you missed to create the DB so you can add a new migrate to add the database for the messages so I highly recommend to add it since I can't see anything in your schema.rb file and no model for the messages 😊🔥 so please create the db for that to make your backend works as expected 💯♻

### After running your app with putting the route it gives me this error ⬇ Kindly check it 💪🏻
![image](https://user-images.githubusercontent.com/58553711/220214209-08680a9b-ef4b-4be2-b4a8-1f027d3f8b35.png)